standards/container_standards.md

```diff
@@ -0,0 +1,49 @@
+# Container standards
+
+## Terminology
+`Dockerfile` - set of instructions for building a docker image  
+`Image` - a constructed set of layered docker instructions  
+`Container` - a running instance of an image
+
+## Standards
+### Images are created using Docker
+### Docker Compose is used for defining container builds
+
+Using a consistent containerisation tool will help enforce standards and better promote developer mobility.
+ 
+### Linux containers are used if the service to be containerised can run on Linux  
+
+Underlying container hosts can only host either Linux or Windows containers.  A model of consistenty would reduce the number of underlying hosts.  Windows containers can only be run on Windows hosts which typically have greater licensing costs vs Linux.
+
+*Note that services built using .Net Framework vs .Net Core cannot be run using Linux containers.  Defra's position is .Net Core should be preferred over .Net Framework but there may be some use cases if containerising legacy services*
+
+### Images are built using Defra base images (see below)
+### Containers are not run using root user
+
+Containers are not trust boundaries and therefore should never be run as root for security reasons.
+
+# Public images are signed
+
+A digital fingerprint should be added to each image.  This enables consumers of images to verify the source and trust levels of the image they are consuming.
+
+### Production images should be immutable and once built do not change but can be configured
```

Comment:
Quick explanation of the rationale, and maybe just a clarification on what exactly we mean by immutable?

Reply:
Rationale added.  Basically the advantage is if the image never changes you can predict it's behaviour across any environment.